### ISSUE TITLE
Docs Review Sign-up

### ISSUE THREAD

**Issue by chrisdickinson** [nodejs/docs#69]
As part of our charter, this WG is required to review documentation coming through PRs in [nodejs/node](https://github.com/nodejs/node). However, we're to **avoid** blocking PRs from landing solely based on documentation concerns.

As a result, I propose we start a review of docs that have landed on the `master` branch of Node over the course of the past week. Our goal will be to clarify, correct, and edit _recently landed_ API docs. This should prevent the Docs WG from blocking Core contributors from landing work, while ensuring a high quality of documentation for releases. Docs to be reviewed will be reported in the Docs WG Slack.

**If your interests lie in spelling, grammar, or editing, this is a task that could benefit from your skills.** Please comment on this issue if you're interested in helping out with this.

**Comment by Knighton910** [comment]
sounds good Chris @chrisdickinson, I'm down for proof reading.

**Comment by techjeffharris** [comment]
> Docs to be reviewed will be reported in the Docs WG Slack.

Is the #github-announce channel integration working in any way/shape/form ATM or are these docs to be reviewed being listed manually until we get integration hooked up?

**Comment by chrisdickinson** [comment]
Right now the listing will be manual — I believe @distracteddev is working on an integration for the channel.

**Comment by thefourtheye** [comment]
It would be better if only the issues or PRs with doc label are notified in the Slack.

**Comment by chrisdickinson** [comment]
@thefourtheye Tagged issues and PRs will likely also be included in the github-announce channel. Only announcing labeled items wouldn't be sufficient, however: core PRs _may_ be opened and landed without the `doc` label being added, or being blocked on Docs WG review (this is by design — one of the concerns the CTC expressed was that Docs WG work shouldn't block core work.)

**Comment by thefourtheye** [comment]
You mean CTC doesn't expect an LGTM from Docs WG to land a doc change?

**Comment by chrisdickinson** [comment]
@thefourtheye In the context of a PR that is primarily updating code, that is correct. If a PR _only_ contains documentation, or is primarily documentation-focused, the Docs WG is expected to LGTM. We may be cc'd on an code-related PR to take a look at the docs, but we're not to hold up merges on those PRs. This is a compromise, but a workable one: this issue sets up part of the process for reviewing merged docs.

**Comment by thefourtheye** [comment]
@chrisdickinson Hmmm okay. What I am worried about is, there may be additional work if something, that doesn't align with Doc WG way of doing things lands then we may have to amend it with another PR. Perhaps we can improve the tools so that the doc changes are linted when `make lint` is done.

**Comment by chrisdickinson** [comment]
@thefourtheye Yep — looping doc linting into `make test` is part of the [number :one: task on the proposed `ROADMAP`](https://github.com/nodejs/docs/blob/add-roadmap/ROADMAP.md#subtasks)!

**Comment by thefourtheye** [comment]
@chrisdickinson Awesome :-) Let me see if I can contribute to that.

**Comment by jinglish** [comment]
> spelling, grammar, or editing

This is my jam.

**Comment by Trott** [comment]
Closing as this repository is dormant and likely to be archived soon. If this is still an issue, feel free to open it as an issue on the [main node repository](https://github.com/nodejs/node).
